service-tests/jetbrains.js

```diff
@@ -93,6 +154,32 @@ t.create('missing required XML attribute')
                  })
   .expectJSON({ name: 'downloads', value: 'invalid' });
 
+t.create('missing required XML attribute(v)')
```

Comment:
You do not need this test. Version value is in `version` element.

Reply:
Fixed. How was that now?